rskj-core/src/main/resources/reference.conf

```diff
@@ -0,0 +1,237 @@
+peer {
+    # Boot node list
+    # Use to connect to specific nodes
+    active = [ ]
+
+    # list of trusted peers the incoming connections is always accepted from. Even if the max amount of connections is reached
+    # This is used to create a filter of Trusted peers
+    trusted = [
+        # Sample entries:
+        # {nodeId = "e437a4836b77ad9d9ffe73ee782ef2614e6d8370fcf62191a6e488276e23717147073a7ce0b444d485fff5a0c34c4577251a7a990cf80d8542e21b95aa8c5e6c"},
+        # {ip = "11.22.33.44"},
+        # {ip = "11.22.33.*"},
+        # {
+        #   nodeId = "e437a4836b77ad9d9ffe73ee782ef2614e6d8370fcf62191a6e488276e23717147073a7ce0b444d485fff5a0c34c4577251a7a990cf80d8542e21b95aa8c5e6c"
+        #   ip = "11.22.33.44"
+        # }
+    ]
+
+    # connection timeout for trying to connect to a peer [seconds]
+    connection.timeout = 2
+
+    # the parameter specifies how much time we will wait for a message to come before closing the channel
+    channel.read.timeout = 30
+
+    # Private key of the peer
+    # nodeId = <NODE_ID>
+    # privateKey = <PRIVATE_KEY>
```

Comment:
shouldn't be uncommented?

Reply:
Not really, we are using the existing feature of auto-generating a `nodeId/PK`